pkgs/development/libraries/amdvlk/default.nix

```diff
@@ -21,13 +21,13 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "amdvlk";
-  version = "2021.Q1.5";
+  version = "2021.Q1.6";
 
   src = fetchRepoProject {
     name = "${pname}-src";
     manifest = "https://github.com/GPUOpen-Drivers/AMDVLK.git";
     rev = "refs/tags/v-${version}";
```

Comment:
```suggestion
    rev = "v-${version}";
```

Reply:
Changing it to `"v-${version}"` fails:
```
Downloading manifest from https://github.com/GPUOpen-Drivers/AMDVLK.git
fatal: couldn't find remote ref refs/heads/v-2021.Q1.6
```

(note that this is `fetchRepoProject`, not `fetchFromGitHub` or something similar)